pkgs/development/tools/supabase-cli/default.nix

```diff
@@ -9,16 +9,16 @@
 
 buildGoModule rec {
   pname = "supabase-cli";
-  version = "1.88.0";
+  version = "1.97.1";
 
   src = fetchFromGitHub {
     owner = "supabase";
     repo = "cli";
     rev = "v${version}";
-    hash = "sha256-OgeKr0xSWp83Ri3WBNsY3TL6tAWsMtsEdD6KeKdPdCw=";
+    hash = "sha256-k1/6QAxQ3h5xG+2VBc/PillddYhrvsQrtJK5tXrb0sQ=";
   };
 
-  vendorSha256 = "sha256-efcgpxvhHe6KtNfNYYc5fYv93fJPf63V39d+5AcCvPQ=";
+  vendorSha256 = "sha256-oEIRmkOCHIvIQJX7GQVJD/aURoEJ7V/LJFEg+oltWEY=";
```

Comment:
```suggestion
  vendorHash = "sha256-oEIRmkOCHIvIQJX7GQVJD/aURoEJ7V/LJFEg+oltWEY=";
```

Reply:
Wouldn't this be better changed upstream on the bot?

This was autogenerated by https://github.com/ryantm/nixpkgs-update

Reply:
https://github.com/ryantm/nixpkgs-update/issues/295#issuecomment-1069124567

Mentions that not every fetcher supported hash in 2022 so the update script kept whichever was used in the derivation. Could have changed since then.

Reply:
https://github.com/NixOS/nixpkgs/pull/254879

This should address that concern treewide